drupal-user-guide/index.rst

```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
 If some templates have been overridden, please compare with the new original one.
 
 If you have defined a sub-theme of obiba_bootstrap's theme, you might need to update your style sheet.
+
+Drupal 7.72
+-----------
+
+For the security issues you should update your Drupal Installation to 7.72 Latest version, make sure to use the bootstrap-7.x-22 version.
```

Comment:
Suggestion:

For the security issues you should update your Drupal installation to the latest version 7.72. Also,  make sure to use the bootstrap-7.x-22 version.